indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -17,11 +17,12 @@
 from __future__ import unicode_literals
 
 from datetime import date, datetime, time, timedelta
+from io import BytesIO
 
 from flask import jsonify, request, session
 from sqlalchemy.orm import subqueryload
 from webargs import fields
-from webargs.flaskparser import use_args, use_kwargs
+from webargs.flaskparser import abort, use_args, use_kwargs
```

Comment:
You're not using it anymore!

Reply:
unrelated, but `abort` is from flask and if used it should be imported from there and not from some library that also imports it (probably also from flask)